generators/app/templates/src/StarterKit/Middleware/RequestResponseLoggingMiddleware.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Serilog;
+using Serilog.Events;
+
+namespace StarterKit.Middleware
+{
+    public class RequestResponseLoggingMiddleware
+{
+    private readonly RequestDelegate _next;
+
+    public RequestResponseLoggingMiddleware(RequestDelegate next)
+    {
+        _next = next;
+    }
+
+    public async Task Invoke(HttpContext context)
+    {
+        var sw = new Stopwatch();
+        sw.Start();
+        //First, get the incoming request
+        await LogRequest(context.Request);
```

Comment:
Misschien ook via een config-key kunnen laten instellen of inkomende requests en of uitgaande reponses moeten gelogd worden?

https://bitbucket.antwerpen.be/projects/ACPAAS/repos/authz_api_aspnetcore/browse/src/AuthzApi/Framework/Logging/IncomingRequestLogger.cs?until=00263e3a1b1bd3f43a11fceb11efe649b0dec23b&untilPath=src%2FAuthzApi%2FFramework%2FLogging%2FIncomingRequestLogger.cs#46